src/BlazorWebService/BlazorWebService/Program.cs

```diff
@@ -26,6 +31,9 @@
 app.UseStaticFiles();
 app.UseAntiforgery();
 
+// Expose the weather forecast API
+app.MapGet("/api/weather", async ([FromServices] WeatherForecastService wfs) => await wfs.GetWeatherForecastAsync());

```

Comment:
I believe `[FromServcies]` isn't required. Minimal APIs should be able to figure out that this is a service without any annotation.

Reply:
Thanks!